### PR TITLE
Added Chromatic Aberration Doc

### PR DESCRIPTION
**Pull request by tjmgd**
Signed-off-by: T.J. McGrath-Daly <tj.mcgrath.daly@huawei.com>

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Comment by sptramer** [comment]
Blocked by https://github.com/o3de/o3de/pull/7640

File: content/docs/user-guide/components/reference/atom/chromatic-aberration.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+linkTitle: Chromatic Aberration
+title: Chromatic Aberration Component
+description: Open 3D Engine (O3DE) Chromatic Aberration component reference. 
+toc: true
+---
+
+The **Chromatic Aberration** component creates *Chromatic Aberration*, a post-processing effect that simulates the failure of a lens to focus all colors to the same point.
```
Comment (micronAMZN) [inline content/docs/user-guide/components/reference/atom/chromatic-aberration.md:8]: Using the word failure here might suggest this a a bad thing to users who aren't familiar with this concept, as opposed to a visual effect that is sometimes desired.

```suggestion
The **Chromatic Aberration** component is a post-processing effect that simulates a lens that focuses different wavelengths of light at different points, creating fringes of color around edges in the image.
```

File: content/docs/user-guide/components/reference/atom/chromatic-aberration.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
```
Comment (micronAMZN) [inline content/docs/user-guide/components/reference/atom/chromatic-aberration.md:10]: Removing some whitespace
```suggestion
```

File: content/docs/user-guide/components/reference/atom/chromatic-aberration.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+## Provider
+
+[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom)
+
+
```
Comment (micronAMZN) [inline content/docs/user-guide/components/reference/atom/chromatic-aberration.md:15]: Removing some whitespace
```suggestion
```

File: content/docs/user-guide/components/reference/atom/chromatic-aberration.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+## Dependencies
+
+[PostFX Layer component](./postfx-layer)
+
+
```
Comment (micronAMZN) [inline content/docs/user-guide/components/reference/atom/chromatic-aberration.md:20]: Removing some whitespace
```suggestion
```

File: content/docs/user-guide/components/reference/atom/chromatic-aberration.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+## Chromatic Aberration properties
+
```
Comment (micronAMZN) [inline content/docs/user-guide/components/reference/atom/chromatic-aberration.md:23]: Need interface image here.

File: content/docs/user-guide/components/reference/atom/chromatic-aberration.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Enable** | If enabled, activates the effect. | Boolean | `Disabled` |
+| **Overrides - Enabled Override** | If enabled, all component properties will be set to the values specified in the **Overrides** property group. | Boolean | `Enabled` |
+| **Strength** | Controls the magnitude of the color displacement.  | Float: 0.0 - 1.0 | `0.01` |
+| **Blend** | Scales the blending of the effect with the original image, i.e. reduces sharpness. | Float: 0.0 - 1.0 | `0.5` |
```
Comment (micronAMZN) [inline content/docs/user-guide/components/reference/atom/chromatic-aberration.md:29]: Suggest avoiding Latin and abbreviations per the style guide.

```suggestion
| **Blend** | Scales the blending of the effect with the original image which reduces the sharpness of the effect. | Float: 0.0 - 1.0 | `0.5` |
```

**Review by micronAMZN — CHANGES_REQUESTED** [review]
Thanks for your contribution!

**Review by FiniteStateGit — COMMENTED** [review]
Thanks for the contribution.

Would it be possible to add a representative screenshot of the effect?  I think something similar to your screenshot of the exaggerated effect would work well, even better if you could isolate the effect from background noise like the grid and hdri map.

chromatic-aberation

**Comment by rgba16f** [comment]
@tjmgd any update?

**Comment by micronAMZN** [comment]
Moved to development branch, because https://github.com/o3de/o3de/pull/7640 is submitted to development.

**Comment by tjmgd** [comment]
I seem to have made a mess of a rebase! I will repair this first.

**Comment by FiniteStateGit** [comment]
@tjmgd I think the Bloom component page was added to the PR instead of Chromatic Aberration when you rebased.

**Comment by FiniteStateGit** [comment]
Thank you for the contribution @tjmgd! Closing this in favor of https://github.com/o3de/o3de.org/pull/1949.

**Pull request closed without merging**